.travis.yml

```diff
@@ -10,12 +9,15 @@ scala:
 jdk:
   - openjdk8
 
-env:
-  global:
-    - secure: "c3gMioUuhU2iiqmV0jx+mig20j0Fu/MLy/h9OCDQfsMi1PqH1woNospBPhvwjQgTm1Wk7ciC8vmYfc9oHVw5rejjKMAgPTgft3NsTXBXoJ4B1RAWFAm5Fg6ERCwxBC+F61HOroye5QbMI5mhdUNp4rrT4e17VqHcNzyuUrolqdw="
-    - secure: "oGZioPdpRRO6ucL0E26h5wD2+Fyhnnb+l87c0C9KzHvGA+3sndf4ikO1YQ4bKw810VwjwL58BViHNk1IGUP/QKatSdEyETyHC/jqDp/6U73wmy7V7XWR0NBu90yOoOlV0/m69j4h/3jdD3PY0Fhcuh0l+4LIrOK4Oh93fJpl3sM="
+stages:
+  - name: test
+  - name: release
+    if: "((branch = master AND type = push) OR (tag IS present)) AND NOT fork"
```

Comment:
So this releases snapshots of each commit to master? Cool

Reply:
Yep :)

I've struggled a bit understanding what the travis scala does, the build seems better now.

Except for the fact that it's a bit faster, since it does the tests in parallel, is there anything against just doing `sbt +test` with whatever build image including sbt?

Reply:
> is there anything against just doing sbt +test with whatever build image including sbt?

Not that I know of